tests/unit/api/deleteMarker.js

```diff
@@ -1,10 +1,14 @@
-import { versioning } from 'arsenal';
-
+import async from 'async';
```

Comment:
There are existing unit tests for multiObjectDelete / Delete that I could have placed these tests in, but I wanted to create a separate file since I knew the tests for creating the delete markers would share functions and set-up.